src/Microsoft.Data.Analysis/DateTimeComputation.cs

```diff
@@ -0,0 +1,314 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Data.Analysis
+{
+    internal class DateTimeComputation : IPrimitiveColumnComputation<DateTime>
+    {
+        public void Abs(PrimitiveColumnContainer<DateTime> column)
+        {
+            throw new NotSupportedException();
+        }
+
+        public void All(PrimitiveColumnContainer<DateTime> column, out bool ret)
+        {
+            throw new NotSupportedException();
+        }
+
+        public void Any(PrimitiveColumnContainer<DateTime> column, out bool ret)
+        {
+            throw new NotSupportedException();
+        }
+
+        public void CumulativeMax(PrimitiveColumnContainer<DateTime> column)
+        {
+            var ret = column.Buffers[0].ReadOnlySpan[0];
```

Comment:
What if it is empty?

Reply:
I had the same thought when I first saw the PR, so I looked at what the other columns are doing. None of them check for empty here. It's not high priority IMO, so I'm thinking we can fix that for all the columns in a separate PR?

Reply:
Can you log an issue for this? So we remember to do it.

Reply:
https://github.com/dotnet/machinelearning/issues/5836